api/v1beta1/verticadb_conversion.go

```diff
@@ -123,6 +131,14 @@ func convertFromAnnotations(src *v1.VerticaDB) (newAnnotations map[string]string
 		}
 		newAnnotations[key] = val
 	}
+	_, VClusterOpsAnnotationOK := src.Annotations[vmeta.VClusterOpsAnnotation]
+	_, RunNMAInSidecarAnnotationOK := src.Annotations[vmeta.RunNMAInSidecarAnnotation]
+	// If the VClusterOpsAnnotation annotation is set to true or not present, set the RunNMAInSidecarAnnotation
+	// to true if it's not present so that CRs converted from v1 will still run NMA in a sidecar container, which is
+	// the default behavior of v1
+	if (!VClusterOpsAnnotationOK || vmeta.UseVClusterOps(src.Annotations)) && !RunNMAInSidecarAnnotationOK {
```

Comment:
yeah I originally thought it would be clearer if we explicitly check whether the annotation exists rather than relying on the behavior of `UseVClusterOps()` with respect to default value. Given that the behavior of `UseVClusterOps()` is expected to be stable in the long term (v1 API will always by default use vclusterops), it makes sense to not check explicitly here.